CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.152.0, 9 June 2021
+
+- Upgrade and pin Terraform to version 1.0.0
+- Add lockfile support for Terraform
+- Pipenv upgrade
```

Comment:
Would be good to mention the version here, and we also upgraded pip-tools and pip